gitops/addons/charts/gitops-bridge/values.yaml

```
@@ -421,21 +421,23 @@ addons:
   cert_manager:
     enabled: false
     releaseName: cert-manager
-    namespace: '{{.metadata.annotations.cert_manager_namespace}}'
+    namespace: '{{default "cert-manager" (index .metadata.annotations "cert-manager_namespace") }}'
     chart: cert-manager
     repoUrl: https://charts.jetstack.io
     targetRevision: "v1.15.2"
+    annotationsAppSet:
+      argocd.argoproj.io/sync-wave: '-2'
     selector:
       matchExpressions:
         - key: enable_cert_manager
           operator: In
           values: ['true']
     values:
       installCRDs: true
-      serviceAccount:
-        name: '{{.metadata.annotations.cert_manager_service_account}}'
-        annotations:
-          eks.amazonaws.com/role-arn: '{{.metadata.annotations.cert_manager_iam_role_arn}}'
+      # serviceAccount:
+      #   name: '{{.metadata.annotations.cert_manager_service_account}}'
+      #   annotations:
+      #     eks.amazonaws.com/role-arn: '{{.metadata.annotations.cert_manager_iam_role_arn}}'
```

```suggestion
      # serviceAccount:
      #   name: '{{.metadata.annotations.cert_manager_service_account}}'
      #   annotations:
      #     eks.amazonaws.com/role-arn: '{{.metadata.annotations.cert_manager_iam_role_arn}}'
```

Leave this in and set the annotation to empty string to be consistent how we do this for other helm charts.
This way the chart remains portable if user wants to use IRSA.
In this file https://github.com/aws-samples/fleet-management-on-amazon-eks-workshop/blob/riv24/gitops/addons/default/addons/gitops-bridge/values.yaml

Add the following section:

```
  cert-manager:
    values:
      serviceAccount:
        annotations:
          eks.amazonaws.com/role-arn: '' # override with empty string because we are using pod identity not IRSA
```